### DISCUSSION TITLE
[Action Required] Update the Google Search Console API discovery document

### DISCUSSION THREAD

**Discussion by inigopascall** [plausible/analytics#1373] [Self-Hosted Support] [answered]
I have received this email this morning from Google, which lists my self-hosted Plausible instance as being possibly affected:

> Hello Google Search Console API Customer,
> 
> In August and December 2020, we announced that we would be removing support for the Webmasters discovery document.
> 
> Today we are writing to let you know that starting November 1, 2021, projects which have not updated their discovery document will no longer be supported and will stop working.
> 
> If you're querying the Search Console API using an external API library, or querying the Webmasters API discovery document directly, you will need to update your API calls to include the changes described in these step-by-step instructions.
> 
> Your projects listed below have used the Search Console API in the last 90 days and may be affected by this change:
> 
>     Plausible Analytics DO Droplet (plausible-analytics-do-droplet)
> 
> If you have any questions or require assistance, please refer to the Google Search help page.
> 
> Thanks for using the Google Search Console API.

I'm not sure I understand what the change entails; or, if I even need to understand. Can anyone advise whether this means my Plausible instance is going to stop working with Google Search Console after November 1st..?

**Comment by inigopascall** [comment]
Nobody? no-one can comment on this at all, no?

**Comment by Vigasaurus** [comment, marked as answer]
#1371 has more discussion + context

**Reply by inigopascall** [reply]
Thanks @Vigasaurus , missed that.
